phobos/utils/naming.py

```diff
@@ -86,8 +86,8 @@ def safelyName(obj, name, phobostype=None):
     # use custom property if the object.name can not be set properly
     if objectname != name:
         obj[phobostype + '/name'] = name
-    elif phobostype + '/name' in obj:
-        del obj[phobostype + '/name']
+    #elif phobostype + '/name' in obj:
```

Comment:
These two lines are an essential element of the current Phobos version.

Basically every Phobostype is labelled here.
We can't comment out these lines.

I would rather suggest adjusting the DROCK-addon, as we can not provide backwards compatibility from within Phobos (too much happened since January).
In addition, the submodel functionality of Phobos is not the same as submodels in the DROCK-plugin.

Reply:
I don't get why these two lines can even influence the functionality of Phobos. If so, I doubt Phobos has a good concept concerning information handling. What does these two lines do? The phobostype labeling is created in the two lines above the commit. But this labeling is only done if the object name is unequal the phobostype name. The two lines in question remove the phobostype labeling in the other case. So if we comment the two lines the result is, that we can have the phobostype labeling information even if the names are equal, which is the case for the modeling kit of DROCK and we don't need case handling later on. Where and why the hell can Phobos fail if this information is there?